apps/api/src/message-schemas/events/s51-advice.schema.json

```diff
@@ -9,10 +9,6 @@
 			"type": "number",
 			"examples": [1234]
 		},
-		"adviceReference": {
```

Comment:
How come we're removing adviceReference? I think this is a requirement unless something has changed

Reply:
I can't see a column in the database to use for this. Looks like it's expecting something similar in format to the case reference.

Reply:
Ah we do have a reference number which is an integer. Maybe the reference string needs to be constructed like `{caseReference}-{advice reference number}`

Reply:
Ok I've just updated it. There is a function already that formats an advice reference.